templates/bake/Model/table.twig

```diff
@@ -45,7 +45,7 @@ class {{ name }}Table extends Table
 
 {%- if primaryKey %}
     {%- if primaryKey is iterable and primaryKey|length > 1 %}
-        $this->setPrimaryKey([{{ Bake.stringifyList(primaryKey, {'indent': false})|raw }}]);
+        $this->setPrimaryKey({{ Bake.exportVar(primaryKey, 0, constant('Brick\\VarExporter\\VarExporter::INLINE_NUMERIC_SCALAR_ARRAY'))|raw }});
```

Comment:
Is there any way we can make this config option easier to use? Not a great example for people to follow.

Reply:
Yes it's not pretty but the `$options` arg of `VarExporter::export()` is a bitmask and using constants is better than arbitrary numbers. I am open to suggestions to make it easier / more readable.

Reply:
We could add globals for the config constants.

Reply:
Another idea would be supporting string arguments and convert them to constants in exportVar().

Reply:
Couldnt it default to a reasonable default in the cake wrapper method?

Reply:
I added a new `exportArray()` with required defaults for inlined arrays.

Reply:
I don't know anything about the options but definitely looks better now.